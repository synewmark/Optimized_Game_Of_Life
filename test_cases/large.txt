version https://git-lfs.github.com/spec/v1
oid sha256:eb13e5bc5fb30de4805407751ff0d752b1b7b0d8e8baab208b62094e8475a375
size 200040021
