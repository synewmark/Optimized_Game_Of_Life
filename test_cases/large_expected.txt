version https://git-lfs.github.com/spec/v1
oid sha256:b51113b6e87c2335fee95b2e5fd5bbac1f7683da8037eff3f4102f1b4e6cd5c7
size 200020000
